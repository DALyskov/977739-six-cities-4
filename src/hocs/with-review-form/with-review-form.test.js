import React from 'react';
import renderer from 'react-test-renderer';

import withReviewsForm from './with-review-form.jsx';

const MockComponent = () => {
  return <div></div>;
};

const MockComponentWrapped = withReviewsForm(MockComponent);

describe(`withReviewsForm_snapchots`, () => {
  it(`withReviewsForm_is_rendered_correctly`, () => {
    const tree = renderer.create(<MockComponentWrapped />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
