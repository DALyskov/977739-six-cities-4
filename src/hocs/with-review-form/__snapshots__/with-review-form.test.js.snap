// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`withReviewsForm_snapchots withReviewsForm_is_rendered_correctly 1`] = `<div />`;
